app/models/manageiq/providers/azure/cloud_manager/provision/cloning.rb

```diff
@@ -50,10 +50,16 @@ def prepare_for_clone_task
     nic_id = associated_nic || create_nic
 
     # TODO: Ideally this would be a check against source.storage or source.disks
-    if source.ems_ref.starts_with?('/subscriptions')
+    if source.ems_ref =~ /.+:.+:.+:.+/
```

Comment:
Does it need to be this tricky? It could just be `.starts_with("http")?` couldn't it?